src/lib/cyleaflet_clientside.js

```diff
@@ -87,4 +95,43 @@ window.dash_clientside.cyleaflet = {
             return e;
         });
     },
+    updateLonLat: function (elements) {
+        if (elements.length > 0) {
+            return elements.map((e) => {
+                if (
+                    typeof e.position !== 'undefined' &&
+                    Object.prototype.hasOwnProperty.call(e.position, 'x') &&
+                    e.position.x !== 0 &&
+                    e.position.y !== 0
+                ) {
+                    var lonLat = xYToLonLat(e.position.x, e.position.y);
+                    return {
+                        position: e.position,
+                        data: Object.assign({}, e.data, {
+                            lon: lonLat[0],
+                            lat: lonLat[1],
+                        }),
+                    };
+                }
+                return e;
+            });
+        }
+        return window.dash_clientside.no_update;
+    },
+    updateCytoMaxZoom: function (children) {
+        var tileLayer = children;
+        if (children.length >= 1) {
+            tileLayer = children.filter((e) => {
+                return e.type === 'TileLayer';
+            })[0];
+        }
```

Comment:
Does this work even if `children` is a list of length 1? I think you'll end up with an extra level of nesting in that case.

Reply:
It works when `children` is of length 1 but I'll add a condition in case TileLayer is not passed in `children`.